scripts/devops_tasks/tox_harness.py

```diff
@@ -291,6 +346,8 @@ def execute_tox_serial(tox_command_tuples):
         if result is not None and result != 0:
             return_code = result
 
+        collect_log_files(cmd_tuple[1])
```

Comment:
This should be within `in_ci` I think.

Reply:
We don't currently move coverage files around etc unless that's the case at least. I could be talked down from that :)